standalone/include/psen_scan_v2_standalone/protocol_layer/scanner_state_machine.h

```diff
@@ -235,10 +235,10 @@ class ScannerProtocolDef : public msm::front::state_machine_def<ScannerProtocolD
   // LCOV_EXCL_STOP
   void checkForInternalErrors(const data_conversion_layer::scanner_reply::Message& msg);
 
-  void checkForDiagnosticErrors(const data_conversion_layer::monitoring_frame::Message& frame);
-  void informUserAboutTheScanData(const data_conversion_layer::monitoring_frame::Message& frame);
-  void sendMessageWithMeasurements(const std::vector<data_conversion_layer::monitoring_frame::Message>& frames);
-  bool framesContainMeasurements(const std::vector<data_conversion_layer::monitoring_frame::Message>& frames);
+  void checkForDiagnosticErrors(const data_conversion_layer::monitoring_frame::Message& msg);
+  void informUserAboutTheScanData(const data_conversion_layer::monitoring_frame::MessageStamped& frame);
+  void sendMessageWithMeasurements(const std::vector<data_conversion_layer::monitoring_frame::MessageStamped>& frames);
+  bool framesContainMeasurements(const std::vector<data_conversion_layer::monitoring_frame::MessageStamped>& frames);
```

Comment:
We should decide whether we ues `frame` or `msg`, here.

Reply:
You are right. The idea was stamped ones are `frames` and without its `msg` but thinking about it, it was probably not the best idea.

Reply:
I remember, why I used frame. Because if you name it msg_stamped you have a lot of msg_stamped.msg_... calls and that just doesn't read as easy as frame.msg_ :/